ztp/policygenerator/policyGen/policyHelper.go

```diff
@@ -58,3 +58,19 @@ func CheckNameLength(namespace string, name string) error {
 	}
 	return nil
 }
+
+// Create a new ObjectTemplate for the given resource with values
+// populated from the reference ACM template (pulled from
+// policyBuilder)
+func BuildObjectTemplate(resource generatedCR) (utils.ObjectTemplates, error) {
```

Comment:
I guess u missed to clean this func. There is no error handling why return an error ? please remove the error return OR if there is an error handle it then return it.